PointOfSaleTerminal/Entities/VolumeDiscountRule.cs

```diff
@@ -6,11 +6,10 @@
 
 namespace PointOfSaleTerminal.Entities
 {
-    public class VolumeRebateRule
+    public class VolumeDiscountRule
```

Comment:
I think we can remove volume at all, and here we see not a rule it is just discount entity.
So better to name it just Discount

Reply:
Done.